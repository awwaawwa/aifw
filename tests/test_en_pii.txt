Please translate the following to Chinese:

Dear Support Team,

This is John A. Doe from Acme Corporation (Username: johndoe_1984). I'm reaching out about a small account issue‚Äîsorry if this message is a bit long üôè. 

My home address is: 1234 Elm Street, Suite 56, Springfield, IL 62704 (yep, the old adress‚Äîspelled wrong on my last form, my bad!).
You can reach me at my email: test.user+alias@example.com, or call me at +1 415-555-2671. When I'm in China, my mobile is 18744325579 (still active).

For the refund you mentioned, my bank account number is 1234 5678 9012 3456. Alternatively, you can use my payment card (dummy for testing only): Visa 4242-4242-4242-4242, CVV 123, exp 12/34 ‚Äî please DO NOT actually charge this; it's just a placeholder for your QA case.

To log in to the staging portal, use this temporary verification code: 9F4T2A. For the sandbox box, the pwd: S3cure!Passw0rd (I'll reset it after your tests, promise!).

For our wallet demo (again, test data only), here's the seed phrase: river apple orange cable window magnet winter fee bonus ladder camera peach.
And below is a dummy private key block (not real, just for parsing checks):
-----BEGIN PRIVATE KEY-----
MIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQC1w8P1x0kQbZpx
uH7u2/1aYgH0b8oE4R2H3yV2gJg0f2oTg9zZQ97lP0JqR9x8Xx4j6ya2q4Z3Xx2F
m2T1z+qk0a5Dq7mWkKX8rJq6fQIDAQABAoIBAQCd9E2J4K1Y9uRFGk1V3k1kGm3T
l9aZKqv0o4h5zY+G7n8Gg3PzKj3e3lG7K1f5m0zV3Y1gM3s1V9r7YjX2Z2c9uL8k
-----END PRIVATE KEY-----

If you need more info, see https://example.com/support ‚Äî or our docs at www.example.org/guide?lang=en-US (sometimes the intranet link http://intranet.local/login redirects weirdly, FYI).

Thanks a ton for your help! If anything looks off, feel free to ping me back.

Sincerely,
John
