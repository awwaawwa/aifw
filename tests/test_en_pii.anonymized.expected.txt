Please translate the following to Chinese:

Dear Support Team,

This is __PII_PERSON_00000001__ from __PII_ORGANIZATION_00000002__ (Username: __PII_USER_NAME_00000003__). I'm reaching out about a small account issue‚Äîsorry if this message is a bit long üôè. 

My home address is: __PII_PHYSICAL_ADDRESS_00000004__ (yep, the old adress‚Äîspelled wrong on my last form, my bad!).
You can reach me at my email: __PII_EMAIL_ADDRESS_00000005__, or call me at __PII_PHONE_NUMBER_00000006__. When I'm in China, my mobile is __PII_PHONE_NUMBER_00000007__ (still active).

For the refund you mentioned, my bank account number is __PII_BANK_NUMBER_00000008__. Alternatively, you can use my payment card (dummy for testing only): Visa __PII_PAYMENT_00000009__, CVV 123, exp 12/34 ‚Äî please DO NOT actually charge this; it's just a placeholder for your QA case.

To log in to the staging portal, use this temporary verification code: __PII_VERIFY_CODE_0000000a__. For the sandbox box, the pwd: __PII_PASSWORD_0000000b__ (I'll reset it after your tests, promise!).

For our wallet demo (again, test data only), here's the seed phrase: __PII_RANDOM_SEED_0000000c__.
And below is a dummy private key block (not real, just for parsing checks):
__PII_PRIVATE_KEY_0000000d__

If you need more info, see __PII_URL_0000000e__ ‚Äî or our docs at __PII_URL_0000000f__ (sometimes the intranet link __PII_URL_00000010__ redirects weirdly, FYI).

Thanks a ton for your help! If anything looks off, feel free to ping me back.

Sincerely,
__PII_PERSON_00000011__
